wheatley/bell.py

```diff
@@ -3,14 +3,16 @@
 errors related to deciding whether the treble is bell #0 or bell #1.
 """
 
+from typing import Any
+
 BELL_NAMES = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "0", "E", "T", "A", "B", "C", "D"]
 
 
 class Bell:
     """ A class to encapsulate the idea of a bell. """
 
     @classmethod
-    def from_str(cls, bell_str: str):
+    def from_str(cls, bell_str: str) -> Any:
```

Comment:
OK that seems like a good idea.  I wasn't super sure what to do because I'm not 100% sure about how the static type checker works.  I'll fix all the class methods to return the base class.

Reply:
Hmmm I get an error saying that `Bell` is not defined....

Reply:
try `'Bell'`

Reply:
or
```
from __future__ import annotations
```

Reply:
Done!